src/request/repository/converter/request-converter.js

```diff
@@ -139,19 +140,25 @@ var getStatusCodeText = (function() {
 })();
 var setupIndex = function(request, type, payload) {
     if (type == 'begin-request') {
-        request._requestStartTime = payload.requestStartTime;
-        request._requestMethod = payload.requestMethod;
-        request._requestUrl = (payload.requestPath || '') + (payload.requestQueryString || '');
+        var url = parse(payload.url);
+        payload.path = url.pathname;
+        payload.query = url.query;
+        
+        request._requestStartTime = payload.startTime;
+        request._requestMethod = payload.method;
+        request._requestUrl = payload.path + payload.query;
         request._requestIsAjax = payload.requestIsAjax;
     }
     if (type == 'end-request') {
-        request._responseStatusCode = payload.responseStatusCode; 
-        request._responseStatusText = getStatusCodeText(payload.responseStatusCode);
-        request._responseContentType = payload.responseContentType;
-        request._responseContentCategory = getContentTypeCategory(payload.responseContentType);  
+        var contentType = payload.headers && payload.headers['Content-Type'];
```

Comment:
For the time being, I'm going to get this working by just pulling up this one property in a case insensitive way. Once we get more use cases where we need multiple header values to be processed we can reevaluate - as copying the object has other implications when it comes to local storage, etc.